lib/structures/Member.js

```diff
@@ -88,7 +88,9 @@ class Member extends Base {
             } else if(data.channel_id || data.mute || data.deaf || data.suppress) {
                 this.guild.voiceStates.update(data);
             }
-            data = data.member;
+            if(data.hasOwnProperty("member")){
```

Comment:
a very small style nit I just noticed, there should be a space between the if condition and the block inside
```suggestion
            if(data.hasOwnProperty("member")) {
```